test/specs/vue.ts

```diff
@@ -1,9 +1,12 @@
 import assert = require('power-assert')
 import sinon = require('sinon')
-import Vue = require('vue')
+import Vue from 'vue'
 import { module, store, Getters, Mutations } from '../../src'
 
 describe('Vue integration', () => {
+  Vue.config.errorHandler = (err, vm, info) => {
```

Comment:
After 2.2.0, Vue does not throw error by default.
https://vuejs.org/v2/api/index.html#errorHandler

So, `throws if mutate state out of mutations when strict mode` fails without this.